### DISCUSSION TITLE
Show operating system name/icon in prompt?

### DISCUSSION THREAD

**Discussion by yooakim** [starship/starship#3995] [General]
I am a newcomer to Starship. What would be the best way to show an operating system icon/name as in this example (showing Ubuntu icon when using Ubuntu):
<img width="510" alt="image" src="https://user-images.githubusercontent.com/862546/168978834-c29eb619-137d-4047-821b-07c29e12fb77.png">

I have look at modules/prompts but not found out the best way to pickup the os ?

**Comment by matchai** [comment]
You should be able to do it with a `custom` module, as shown here: https://github.com/starship/starship/discussions/1252#discussioncomment-212353
